requirements-test.txt

```diff
@@ -0,0 +1,20 @@
+amazon-braket-pennylane-plugin
+black
+flake8
+flake8-rst-docstrings
+isort
+pre-commit
+pylint
+pytest==7.1.2
+pytest-benchmark
+pytest-cov
+pytest-rerunfailures
+pytest-timeout
+pytest-xdist
+qiskit==1.1.2
```

Comment:
Would be good to set this to the current NBI version of qiskit (1.0.2?)

Reply:
I am looking to get that bumped up in https://github.com/amazon-braket/amazon-braket-examples/pull/614. I will adjust to use `>=` accordingly